triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/TriggerRuleSimulationLibTest.scala

```diff
@@ -19,6 +18,7 @@ import org.scalacheck.Gen
 import org.scalatest.{Inside, TryValues}
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AsyncWordSpec
+import scalaz.syntax.tag._
```

Comment:
Changes in this file all relate to using `Party` rather than `String` as a method argument